### PR TITLE
ClearLogs command

### PR DESCRIPTION
**Pull request by elliotmassen**
This PR adds a command that allows you to remove logs that are older than some specified amount of days.

### CODE REVIEW

File: app/Console/Commands/ClearLogs.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $days = $this->option("days");
+        $thresholdDate = Carbon::now()->subDays($days)->setTime(0, 0, 0, 0);
```
Comment (stu-greenshoots) [inline app/Console/Commands/ClearLogs.php:34]: Maybe validate the number of days here? Just thinking a miss-type might delete more than you thought?

File: app/Console/Commands/ClearLogs.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            $matches = [];
+            $isMatch = preg_match("/^laravel\-(.*)\.log$/i", $file, $matches);
+
+            if (count($matches) > 1) {
+                $date = $matches[1];
+            }
+
+            $key = $isMatch ? $date : "";
+            return [$key => $file];
+        })->forget("");
```
Comment (stu-greenshoots) [inline app/Console/Commands/ClearLogs.php:48]: What does this forget("") do?

Comment (elliotmassen) [reply app/Console/Commands/ClearLogs.php:59]: It removes an element of the collection with the key of "" ([docs](https://laravel.com/docs/5.8/collections#method-forget)). What happens in the map above is that if we can't find a date in the file name we set the key to "", and if this happens multiple times they will overwrite eachother, and then after that we _forget_ that key anyway. Then we are left with a collection of only valid, dated, log files for deletion.

Comment (stu-greenshoots) [reply app/Console/Commands/ClearLogs.php:59]: thanks, makes sense!

**Pull request merged**